ansible/roles/test/files/ptftests/sonic.py

```diff
@@ -469,9 +469,12 @@ def verify_bgp_neigh_state(self, dut=None, state="Active"):
 
     def change_neigh_lag_state(self, intf, is_up=True):
         state = ['shutdown', 'startup']
-        is_match = re.match(r'(Port-Channel|Ethernet)\d+', intf)
+        pattern = r'(PortChannel|Ethernet)\d+'
+        is_match = re.match(pattern, intf)
         if is_match:
             self.do_cmd('sudo config interface %s intf' % state[is_up])
```

Comment:
I just noticed this,  does the `intf` in this string need to be replaced with the actual interface name?

Reply:
Yes, it does. That's another fix I was going to raise in another PR. But I'll actually just add that here.

Reply:
Fixed. Thanks for pointing it out.